Sources/Signal.swift

```diff
@@ -108,16 +108,11 @@ public final class Signal<Value, Error: Swift.Error> {
 		/// Used to ensure that events are serialized during delivery to observers.
 		private let sendLock: Lock
 
-		/// Used to indicate if the `Signal` has deinitialized.
-		private var hasDeinitialized: Bool
-
 		fileprivate init(_ generator: (Observer) -> Disposable?) {
```

Comment:
The above documentation needs to be updated to reflect that the lock must be held to read the state.